runbotics/runbotics-orchestrator-ui/src/hooks/useTranslations.tsx

```diff
@@ -45,4 +45,31 @@ export const translate = (
     values?: { [key: string]: string | number },
 ) => i18next.t(key, values);
 
+export const checkIfKeyExists = (keyPath: string, key: string ) => {
+    const { t, i18n } = useTranslation();
```

Comment:
you cannot use hooks in basic functions, use static object of i18n like in `isNamespaceLoaded`

Reply:
Is this ok? 

`export const checkIfKeyExists = (keyPath: string, key: string ) => {
    const exist = i18next.exists(keyPath + key)

    return exist
}`